app/views/search_form.php

```diff
@@ -10,14 +10,17 @@
         <fieldset id="main_search">
             <div id="search">
                 <p class="smallscreen_notices">Use the <a href="" class="menu-button">Menu tab</a> in the right top corner to select a different view.<p>
-                <input type="text"
-                       name="recherche"
-                       id="recherche"
-                       value="<?=$initial_search?>"
-                       placeholder="Type your search here"
-                       title="Type your search here"
-                       size="30"
-                />
+                <div id="search_field_container">
+                    <input type="text"
+                           name="recherche"
+                           id="recherche"
+                           value="<?=$initial_search?>"
+                           placeholder="Type your search here"
+                           title="Type your search here"
+                           size="30"
+                    />
+                    <span id="clear"></span>
```

Comment:
Not the best in terms of accessibility, but I would add a title like "Clear the search field" to the span.
Also, `clear_search` as ID? `clear` seems a bit too generic.